lua/smart-splits/config.lua

```diff
@@ -147,6 +149,9 @@ function M.setup(new_config)
     config.at_edge = config.wrap_at_edge == true and AtEdgeBehavior.wrap or AtEdgeBehavior.stop
     vim.deprecate('config.wrap_at_edge', "config.at_edge = 'wrap'|'split'|'stop'", 'smart-splits.nvim')
   end
+  if config.wezterm_bin_path then
+    M.wezterm_bin_path = config.wezterm_bin_path
+  end
```

Comment:
This isn't needed because we're doing `config = vim.tbl_deep_extend('force', config, new_config or {})` above (I know, it's a bit confusing because of the way it's set up with the metatable, it's more complicated than it needs to be, but that's another problem for me to fix another time :sweat_smile:) 

```suggestion
```

Reply:
> This isn't needed because we're doing `config = vim.tbl_deep_extend('force', config, new_config or {})` above (I know, it's a bit confusing because of the way it's set up with the metatable, it's more complicated than it needs to be, but that's another problem for me to fix another time 😅)

Actually I asked my mentor to review it and he brought up this exact point and a couple other small things, I ended up over complicating things.
I think I have a relatively better understanding now.